src/nnet3/nnet-compute.h

```diff
@@ -47,6 +48,30 @@ struct NnetComputeOptions {
 
 };
 
+struct NnetComputeState {
```

Comment:
should probably have a comment describing this and explaining that it's only needed for NnetBatchLoopedComputer.